examples/sdk_tests/queue/add_consumer.w

```diff
@@ -0,0 +1,23 @@
+bring cloud;
+
+let q = new cloud.Queue();
+let c = new cloud.Counter();
+
+resource JSHelper {
+  init(){}
+  extern "./helper.js" inflight sleep(milli: num);
+}
+
+q.add_consumer(inflight () => {
+  c.inc();
+});
+
+let js = new JSHelper();
+new cloud.Function(inflight () => {
+  q.push("hello");
+  q.push("world");
+  // TODO: replace this sleep with std.sleep: https://github.com/winglang/wing/issues/1535
+  js.sleep(1000);
+  let count = c.peek();
+  assert(count == 2);
+}) as "test";
```

Comment:
Since now `on_message` became` add_consumer` we don't need to keep both files (as long as I don't miss anything), they can be gathered into one

Reply:
@tsuf239 sorry totally missed this message. Yes you are right about `on_message` I think I have a TODO in that file to delete it once `add_consumer` is done. 

Ill look into the AWS failures shortly

Reply:
cool, I'll remove the `on_message` test for now and will open issues for looking at the errors so that we can merge those tests :) 
Is it ok with you?

Reply:
yes cool with me thank you